cmd/xl-v1-multipart.go

```diff
@@ -797,12 +818,13 @@ func (xl xlObjects) AbortMultipartUpload(ctx context.Context, bucket, object, up
 	}
 	defer uploadIDLock.Unlock()
 
-	if !xl.isUploadIDExists(bucket, object, uploadID) {
-		return errors.Trace(InvalidUploadID{UploadID: uploadID})
+	if !xl.isUploadIDExists(ctx, bucket, object, uploadID) {
+		logger.LogIf(ctx, InvalidUploadID{UploadID: uploadID})
```

Comment:
No need to log here since `isUploadIDExists` already takes care of the `InvalidUploadID` error condition.

Reply:
done